scripts/superdesk-dashboard/module.js

```diff
@@ -1,13 +1,21 @@
 (function() {
     'use strict';
 
-    DashboardController.$inject = ['$scope', 'desks', 'dashboardWidgets', 'api', 'session', 'workspaces', 'modal', 'gettext'];
-    function DashboardController($scope, desks, dashboardWidgets, api, session, workspaces, modal, gettext) {
+    DashboardController.$inject = ['$scope', 'desks', 'dashboardWidgets', 'api', 'session', 'workspaces', 'modal', 'gettext', 'privileges'];
+    function DashboardController($scope, desks, dashboardWidgets, api, session, workspaces, modal, gettext, privileges) {
         var vm = this;
 
         $scope.edited = null;
         $scope.workspaces = workspaces;
         $scope.$watch('workspaces.active', setupWorkspace);
+
+        // canConfigure returns true if the current user is allowed to configure
+        // the active workspace. The active workspace may be configured if it is
+        // a custom workspace or if the user has desk management privileges.
+        $scope.canConfigure = function() {
```

Comment:
I have thought about how to make this function less repetitive and have not came up with a better option. I think it is simple enough to have it as is, although I would also prefer it to be in one place, instead of two.

However, I don't think having it in the `workspaces` service is appropriate. I don't think it would be appropriate for the service to interact with session based data, but if you think otherwise, please explain.